docs/tutorials/intro_to_chemistry.ipynb

```diff
@@ -0,0 +1,310 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Intro To Chemistry Tutorial.ipynb",
+      "provenance": [],
+      "collapsed_sections": []
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "N4327P0aWzVh",
+        "colab_type": "text"
+      },
+      "source": [
+        "## Introduction to Electronic Structure Theory\n",
+        "Electronic Structure Theory describes the motions of electrons in atoms or molecules. Because the electrons are so small, one needs to use quantum mechanics to solve for their motion.\n",
```

Comment:
Nit: 'Electronic structure theory' (no capitalization needed).

Reply:
Just as a general rule - in scientific English we tend to capitalize nouns that are related to people's names (Hamiltonian, Gaussian), and the names of elements (Hydrogen, Helium), and not so much else (except at the start of sentences - and optionally in titles). Typically within a sentence if in doubt don't capitalise.

Reply:
Thanks for the clarification -- tried to fix this throughout. PTAL.